tools/nni_cmd/launcher_utils.py

```diff
@@ -230,6 +230,9 @@ def validate_all_content(experiment_config, config_path):
     validate_pai_trial_conifg(experiment_config)
     experiment_config['maxExecDuration'] = parse_time(experiment_config['maxExecDuration'])
     if experiment_config.get('advisor'):
+        if experiment_config.get('assessor'):
```

Comment:
If advisor is defined, tuner cannot be defined either.

Reply:
Note: In our tuning algorithm's design, the user can choose "Tuner + Assessor(optional) " or "Advisor".

Reply:
> If advisor is defined, tuner cannot be defined either.

Updated, check tuner together with assessor.

Reply:
> Note: In our tuning algorithm's design, the user can choose "Tuner + Assessor(optional) " or "Advisor".

Yes, updated the logic.